drivers/CMakeLists.txt

```diff
@@ -10,3 +10,6 @@ add_subdirectory_ifdef(CONFIG_SENSOR sensor)
 add_subdirectory_ifdef(CONFIG_NETWORKING net)
 add_subdirectory_ifdef(CONFIG_ADP536X adp536x)
 add_subdirectory_ifdef(CONFIG_ST25R3911B_LIB st25r3911b)
+add_subdirectory_ifdef(CONFIG_FLASH flash)
+add_subdirectory_ifdef(CONFIG_CLOCK_CONTROL clock_control)
+add_subdirectory_ifdef(CONFIG_ENTROPY_GENERATOR entropy)
```

Comment:
Let's move these drivers into drivers/bt_ll_nrfxlib/.

Then it will be even more clear that these drivers are specific to bt_ll_nrfxlib and we can encode this pattern once, instead of once for each driver:

```
if(BT_LL_NRFXLIB)
add_subdirectory(bt_ll_nrfxlib)
endif()
```

```
if BT_LL_NRFXLIB
rsource "bt_ll_nrfxlib/Kconfig"
endif
```

Reply:
I don't have very strong opinions on this. Pattern-wise, I like `drivers/<type>/*` better than dumping everything under `bt_ll_nrfxlib`.

Reply:
NB, I meant

drivers/bt_ll_nrfxlib/<type>/